src/spaces/grid_single.jl

```diff
@@ -37,9 +37,12 @@ function GridSpaceSingle(d::NTuple{D,Int}; periodic = true, metric = :chebyshev)
 end
 # Implementation of space API
 function add_agent_to_space!(a::A, model::ABM{<:GridSpaceSingle,A}) where {A<:AbstractAgent}
-    model.space.stored_ids[a.pos...] = a.id
+    pos = a.pos
+    !isempty(pos, model) && error("Attempted to add agent $(a) to occupied position $(pos)")
```

Comment:
```suggestion
    !isempty(pos, model) && error("Cannot add agent $(a) to occupied position $(pos)")
```

Reply:
It seems that all of the `error` messages in `Agents` are in the present tense and say "Cannot" or something similar. This is really minor bikeshedding, so feel free to ignore.

Reply:
This way seems better to me so I'm okay with the change!